src/store/storeConfig.ts

```diff
@@ -284,8 +286,13 @@ export const storeConfig: StoreOptions<State> = {
     },
 
     // update the selected mesh
-    updateSelectedMesh({ commit }, mesh) {
-      ClientStorage.set('selectedMesh', mesh)
+    updateSelectedMesh({ commit }, mesh: string | null) {
+      if (mesh !== null) {
+        ClientStorage.set('selectedMesh', mesh)
+      } else {
+        ClientStorage.remove('selectedMesh')
+      }
+
```

Comment:
Is this a little bugfix for something? Is it worth splitting this out into a separate PR if so?

Reply:
See https://github.com/kumahq/kuma-gui/pull/493.